src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/SemanticTokensCacheTest.cs

```diff
@@ -185,8 +186,8 @@ private static DocumentUri GetDocumentUri(string path)
             fragment: null);
     }
 
-    private static SemanticTokensCache GetSemanticTokensCache()
+    private static DefaultSemanticTokensCacheService GetSemanticTokensCache()
```

Comment:
Nit: If this is specifically a test of `DefaultSemanticTokensCacheService` lets change the file-name.